BE/src/test/java/org/team4/airbnb/util/EntityCreator.java

```diff
@@ -0,0 +1,53 @@
+package org.team4.airbnb.util;
+
+import java.util.ArrayList;
+import java.util.List;
+import org.team4.airbnb.accommodation.Accommodation;
+import org.team4.airbnb.accommodation.AccommodationRepository;
+import org.team4.airbnb.customer.Customer;
+import org.team4.airbnb.customer.CustomerRepository;
+import org.team4.airbnb.wish.Wish;
+
+public class EntityCreator {
+
+	private static String userIdForTest = "test";
+	private static int listSize = 3;
+
+	public static Wish createWish(CustomerRepository customerRepository,
```

Comment:
Utils니까.. `세개의_Wish_생성_요청` 라고 변수명을 바꾸어줍시다.
```suggestion
	public static Wish 세개의_Wish_생성_요청(CustomerRepository customerRepository,
```